### ISSUE TITLE
VS Code + pylint integration with p5

### ISSUE THREAD

**Issue by zaha** [p5py/p5#66]
Hey,

I've installed the lib on my system and copied the demo file. It runs well, but in VS Code with pylint it's giving me quite a lot of errors and warnings like:

- E0102:function already defined line 1 (setup function)
- E0102:function already defined line 1 (draw function)
- E0602:Undefined variable 'mouse_is_pressed'
- W0614:Unused import setup from wildcard import
- and lots more

How  can I set this up to play well with VS Code and pylint?
Sorry if this may sound dumb, I'm pretty new to python :)

**Comment by zaha** [comment]
Great, thanks for the quick response. Hopefully somebody knows :)

btw, great library. I completely agree with your idea that Python is better suited for this than Java. Really hope the missing functionality is added soon as well! 👍

**Comment by jeremydouglass** [comment]
Pylint is configured locally with a .pylintrc file in the root directory of the project being lint-ed.

I encountered a related problem using pylint with Processing.py `.pyde` projects. Here is a gist with the .pylintrc file that I added to projects directories to make them meaningfully lintable. It is largely taken from the Processing API. The config for p5py would be different -- but, I imagine, highly related.

https://gist.github.com/jeremydouglass/bc2193518a29cbf280d0cfc7415a9c82
